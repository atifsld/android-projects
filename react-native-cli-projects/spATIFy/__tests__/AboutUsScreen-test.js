import 'react-native';
import renderer from 'react-test-renderer'
import React from 'react';
import AboutUsScreeen from '../screens/AboutUsScreen';

test('test about us screen', () => {
    const tree = renderer.create(<AboutUsScreeen />).toJSON();
    expect(tree).toMatchSnapshot();
  });