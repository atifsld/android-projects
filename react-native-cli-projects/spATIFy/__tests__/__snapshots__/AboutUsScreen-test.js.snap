// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test about us screen 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "space-between",
      "padding": 26,
    }
  }
>
  <View
    style={
      Object {
        "paddingTop": 32,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../images/logo.png",
        }
      }
      style={
        Object {
          "alignSelf": "center",
          "elevation": 4,
          "height": 128,
          "shadowColor": "#02645b",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.25,
          "shadowRadius": 8,
          "width": 128,
        }
      }
    />
  </View>
  <View
    style={Object {}}
  >
    <Text
      style={
        Object {
          "color": "#0387b4",
          "fontSize": 20,
        }
      }
    >
      Spatify was created to be a catalogue of the world's best music - every week, a selection of 4 albums is listed by celebrated music critiques and musicologists.
    </Text>
  </View>
  <View
    style={Object {}}
  >
    <View
      style={
        Object {
          "paddingBottom": 12,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "black",
            "fontSize": 20,
          }
        }
      >
        Reach out to us!
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 4,
        }
      }
    >
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": "#0387b4",
              "fontSize": 24,
            },
            undefined,
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
      <Text
        style={
          Object {
            "color": "#0387b4",
            "paddingLeft": 10,
          }
        }
      >
        atif.mohd.sld@gmail.com
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 4,
        }
      }
    >
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": "#0387b4",
              "fontSize": 24,
            },
            undefined,
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
      <Text
        style={
          Object {
            "color": "#0387b4",
            "paddingLeft": 10,
          }
        }
      >
        +91 73560 15305
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 4,
        }
      }
    >
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": "#0387b4",
              "fontSize": 24,
            },
            undefined,
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
      <Text
        style={
          Object {
            "color": "#0387b4",
            "paddingLeft": 10,
          }
        }
      >
        atifsld
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
        }
      }
    >
      Version 1.0
    </Text>
  </View>
</View>
`;
